src/main/java/frc/robot/subsystems/BallTransport.java

```diff
@@ -47,6 +51,10 @@ public BallTransport(final BallTransportMap map) {
         this.laserSwitch = map.getLaserSwitch();
     }
 
+    public ColorBufferSupplier getColorBuffer() {
```

Comment:
Prefer returning `colorBuffer` directly, and using `ballTransport::getColorBuffer` as the function reference.